### PR TITLE
Cumulus 2203 -- Release 2.0.8

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-2203: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2203)

## Changes

* Updates Core lambdas to use cma-js 1.3.2
* Version up to 2.0.8

## PR Checklist

- [x] Update CHANGELOG
- [x] Unit tests
- [x] Adhoc testing
- [x] Integration tests

### CODE REVIEW

File: website/versions.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [
+  "v2.0.8",
```
Comment (Jkovarik) [inline website/versions.json:2]: In this instance I triple-verified the version up script did not create a previous doc.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,17 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
-## [v2.0.7] 2020-10-1
+## [v2.0.8] 2020-10-13
+
+### Fixed
+
+- **CUMULUS-2203**
+  - Update Core tasks use cma-js v1.3.2 to resolve memory leak/lambda ENOMEM
```
Comment (markdboyd) [inline CHANGELOG.md:12]: Nit: can we use the actual repo name when referring to CMA JS? `cumulus-message-adapter-js`. Also preferably link to it

File: packages/cmr-client/API.md [hunk 3]
```diff
@@ -12,14 +12,6 @@ CMR search, without loading them all into memory at once.  Handles paging.</p>
 </dd>
 </dl>
 
-## Functions
```
Comment (markdboyd) [inline packages/cmr-client/API.md:15]: This is interesting to me given that there weren't any code changes to this package in between the versions... I'm inclined to trust it, but still

File: CHANGELOG.md (hunk 2 above)
Comment (Jkovarik) [reply CHANGELOG.md]: Sure: e2e1552b7

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,7 +4,19 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
-## [v2.0.7] 2020-10-1
+## [v2.0.8] 2020-10-13
+
+### Fixed
+
+- **CUMULUS-2203**
+  - Update Core tasks use
```
Comment (markdboyd) [inline CHANGELOG.md:12]:
```suggestion
  - Update Core tasks to use
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -3275,7 +3287,8 @@ Note: There was an issue publishing 1.12.0. Upgrade to 1.12.1.
 
 ## [v1.0.0] - 2018-02-23
 
-[unreleased]: https://github.com/nasa/cumulus/compare/v2.0.7...HEAD
+[unreleased]: https://github.com/nasa/cumulus/compare/v2.0.8...HEAD
+[v2.0.8]: https://github.com/nasa/cumulus/compare/v2.0.6...v2.0.8
```
Comment (markdboyd) [inline CHANGELOG.md:3291]:
```suggestion
[v2.0.8]: https://github.com/nasa/cumulus/compare/v2.0.7...v2.0.8
```

Comment (Jkovarik) [reply CHANGELOG.md:3291]: 🤦

File: packages/cmr-client/API.md (hunk 3 above)
Comment (Jkovarik) [reply packages/cmr-client/API.md:15]: Indeed - it wasn't updated in a prior release.

**Review by markdboyd — APPROVED** [review]

**Pull request merged**